core/src/main/java/com/dooapp/fxform/model/impl/AbstractFieldElement.java

```diff
@@ -50,4 +85,48 @@ public String getName() {
 	public Class getDeclaringClass() {
 		return field.getDeclaringClass();
 	}
+
+	@Override
+	protected Binding<ObservableValue<WrappedType>> createValue() {
+		if (List.class.isAssignableFrom(getType())) {
+			return new ListBinding() {
+				{
+					super.bind(sourceProperty());
+				}
+
+				@Override
+				protected ObservableList computeValue() {
+					if (getSource() == null) {
+						return null;
+					}
+					return (ObservableList) AbstractFieldElement.this.computeValue();
+				}
+
+				@Override
+				public void dispose() {
+					super.dispose();
+					unbind(sourceProperty());
+				}
+			};
+		}
+		return new ObjectBinding<ObservableValue<WrappedType>>() {

```

Comment:
Could you add an **else** statement for better readability here ?

Reply:
I can, but I'm from the side of people that think it's a better readability without the else when not needed in case of return statement :
- https://www.refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html
- https://stackoverflow.com/questions/3261849/why-is-else-rarely-used-after-if-x-then-return